src/widget/wtracktableview.cpp

```diff
@@ -505,6 +502,10 @@ void WTrackTableView::slotShowHideTrackMenu(bool show) {
 }
 
 void WTrackTableView::contextMenuEvent(QContextMenuEvent* event) {
+    TrackModel* pTrackModel = getTrackModel();
+    if (!pTrackModel) {
+        return;
+    }
```

Comment:
trackmodel is unused here. Do we really need to check it then?

Reply:
Hm, not directly. But this is called by `[Library], show_track_menu`, so this is one function a controller could call before we have track model (or just item model).

Tbh I'm not entirely happy with this PR but it's the best / most simple fix I found.
There are two aspects:
1. verify we have a selectionModel()
_could_ happen there is none and we want to throw a warning then, eg. it's theoretically_ possible to `setSelectionModel(nullptr)`
2. verify we have a model and simply return if there's none
that's the controller case I mentioned, and (again _theoretically_) it is possible such functions in other ways before the GUI is ready.
I think we should not throw an assertion then. Admittedly this only affects builds with DEBUG_ASSERTIONS_FATAL, but nevertheless such assertions are hard to debug.

Reply:
But I can remove it here, since it's in getSelectedRows(), too.

Reply:
yeah, remove it when `getSelectedRows()` already takes care of it.